db/db.go

```diff
@@ -58,7 +59,8 @@ type db struct {
 
 	events events.Events
 
-	parser core.Parser
+	parser        core.Parser
+	lenseRegistry client.LensRegistry
```

Comment:
:gb: I'll go through and make sure these are consistent later as I'll probably accidentally use the more sensible spelling a few times between now and review time.  As painful as it might be, `Lens` the product/repo uses the Simplified English variant and the internal code should ideally reflect that.